src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -147,7 +147,7 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, $value = nul
      *
      * @return ChoiceListView The choice list view
      */
-    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null)
+    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null, $labelAttr = null)
```

Comment:
Well, the issue is that we cannot make such a BC break in a minor release. So we should find another solution that would be mergable for the 3.1 release (I don't have any idea yet).